### PR TITLE
[WIP] - Model structures

### PR DESCRIPTION
**Pull request by archietype**

### CODE REVIEW

**Comment by kkz6** [comment]
remove all the softdeletes for all the model. @Sreerajsrrk 
also we don't need any factories as well. 

All these data are from click up. which in future might change. So better not to have them

File: app/Http/Controllers/ClickupController.php
Comment (kkz6) [inline app/Http/Controllers/ClickupController.php:1]: We need to remove this controller. We need to sync clickup data with cron job. So we need to create command and sync data

File: app/Models/Team.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Team extends Model
+{
+    use SoftDeletes, HasFactory;
```
Comment (kkz6) [inline app/Models/Team.php]: remove these traits from all models

File: database/migrations/2024_05_14_041225_create_tasks_table.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+use App\Models\Team;
+use App\Models\User;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    public function up(): void
+    {
+        Schema::create('tasks', function (Blueprint $table) {
```
Comment (kkz6) [inline database/migrations/2024_05_14_041225_create_tasks_table.php]: every table name and model name to be something like clickup user , ClickUpUsers. Modify all

**Review by kkz6 — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/ClickupController.php
Comment (archietype) [reply app/Http/Controllers/ClickupController.php:1]: updated

File: database/migrations/2024_05_14_041225_create_tasks_table.php (hunk 2 above)
Comment (archietype) [reply database/migrations/2024_05_14_041225_create_tasks_table.php]: updated

File: app/Models/Team.php (hunk 1 above)
Comment (archietype) [reply app/Models/Team.php]: updated

**Comment by archietype** [comment]
@kkz6 Tasks table relation is still pending, let me know when you're free, have some doubts in it.

**Review by kkz6 — APPROVED** [review]

**Pull request merged**